Cargo.lock

```diff
@@ -76,6 +76,52 @@ version = "1.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "d468802bab17cbc0cc575e9b053f41e72aa36bfa6b7f55e3529ffa43161b97fa"
 
+[[package]]
+name = "axum"
```

Comment:
this shouldn't be necessary. we probably need to turn off default features somewhere